modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -1068,7 +1068,7 @@ public DefaultCodegen() {
         typeMapping.put("file", "File");
         typeMapping.put("UUID", "UUID");
         typeMapping.put("URI", "URI");
-        //typeMapping.put("BigDecimal", "BigDecimal"); //TODO need the mapping?
+        typeMapping.put("BigDecimal", "BigDecimal"); //TODO need the mapping?
```

Comment:
@siilats why do you need to uncomment the mapping in the default codegen as it's already done in Swfit4Codegen: https://github.com/OpenAPITools/openapi-generator/blob/7e58f0325bfdafda2edad1af7603440808fc763a/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/Swift4Codegen.java#L203 ?

Reply:
If you remove this, I'm happy to merge the PR and discuss the mapping of BigDecimal in the default codegen if needed.